app/models/item.rb

```diff
@@ -7,4 +7,17 @@ class Item < ApplicationRecord
   belongs_to :user
   has_many :order_items
   has_many :orders, through: :order_items
+
+  def average_fulfilled_time
+    fulfillment = Item.joins(:orders)
```

Comment:
Do we need to join :orders? If we are only using Item and order_items attributes, could we just use `Item.joins(:order_items)`